src/main/java/Davvy.java

```diff
@@ -0,0 +1,110 @@
+import java.util.Scanner;
+
+import static java.lang.Integer.parseInt;
+
+public class Davvy {
+    private static final String BOT_NAME = "Davvy";
+    private static boolean toEnd;
```

Comment:
Suggest better naming with a more boolean-like name, e.g. `isExitStatus`

Reply:
Thank you for the suggestion!